sdcm/cluster.py

```diff
@@ -1792,6 +1798,23 @@ def node_setup(node):
         time_elapsed = time.time() - start_time
         self.log.debug('Setup duration -> %s s', int(time_elapsed))
 
+    def get_monitor_snapshot(self):
+        for n, node in enumerate(self.nodes):
+            try:
+                process.run('wget "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2"',
+                            verbose=True)
+                process.run('tar xvfj phantomjs-2.1.1-linux-x86_64.tar.bz2',
```

Comment:
Maybe we could reduce verbosity in all uncompressing commands, as the verbose output doesn't help a whole lot. This is just me thinking out loud, no action would be necessary here.